src/view/screens/Debug.tsx

```diff
@@ -177,14 +177,17 @@ function ErrorView() {
 }
 
 function NotifsView() {
+  const {_} = useLingui()
   const triggerPush = () => {
     // TODO: implement local notification for testing
   }
   const triggerToast = () => {
-    Toast.show('The task has been completed')
+    Toast.show(_(msg`The task has been completed`))
```

Comment:
@ansh I'm confused because there are already localized strings in the `Debug.tsx` file, should we don't localize them all in `Debug.tsx`?

Reply:
Let's note that I didn't want to translate all the "fake" text in most of this page, for obvious reasons (it's a debug page, and translating it won't add much value), but translating those two Toasts is not that much work and would make sure we could maybe check later through CI (or types?) that we are always translating toasts.